packages/iov-cli/README.md

```diff
@@ -108,26 +107,25 @@ const sendTx: SendTransaction = {
 > profile.wallets.value
 [ { id: 'ReYESw51lsOOr8_X', label: undefined } ]
 
-> const wallet2 = profile.addWallet(Ed25519HdWallet.fromMnemonic("organ wheat manage mirror wish truly tool trumpet since equip flight bracket"))
+> const wallet2 = profile.addWallet(Secp256k1HdWallet.fromMnemonic("organ wheat manage mirror wish truly tool trumpet since equip flight bracket"));
```

Comment:
Are the semicolons important in this context?

Reply:
No, not at all. I have very mixed feelings about their presence and absence in a REPL tutorial. This is the reson why they are very inconsistent in that file. Any preference?

Reply:
I'll remove the semicolon from the diff for now since there is no reason for the change. But I'm still interested in your preference for future updates

Reply:
I have no strong preference besides consistency. I'd mildly prefer no semicolons because it's more true to actual usage - who bothers typing semicolons in a REPL?